docs/modules/deploy/pages/deploying-in-kubernetes.adoc

```diff
@@ -1,12 +1,21 @@
-= Kubernetes/OpenShift Deployment
+= Deploying a Cluster on Kubernetes/OpenShift
```

Comment:
This page is very long. And the Kubernetes itself is getting more and more popular, so I guess the documentation about Kubernetes will be longer and longer. How about splitting this page into multiple ones. Currently we have:
```
Deploying a Cluster on Kubernetes/OpenShift
  - Deploy a Cluster with the Enterprise Operator (Preview)
```

How about changing it to the following?
```
Kubernetes
  - Deploying a Cluster on Kubernetes/OpenShift => From start to "Discovering Members Automatically"
  - Configuring Hazelcast                                       => From "Discovering Members Automatically" to end
  - Deploy a Cluster with the Enterprise Operator (Preview)
```

What do you think?

Reply:
Looks good. I think we can also separate Scaling and Upgrading Clusters into its own page.